scripts/freebsd/vmtools.sh

```diff
@@ -44,7 +44,7 @@ vmware-iso|vmware-vmx)
     mdconfig -a -t vnode -f $HOME_DIR/freebsd.iso -u 0;
     mount -t cd9660 /dev/md0 /tmp/vmfusion;
     tar xzf /tmp/vmfusion/vmware-freebsd-tools.tar.gz -C /tmp/vmfusion-archive;
-    /tmp/vmfusion-archive/vmware-tools-distrib/vmware-install.pl --force-install;
+    /tmp/vmfusion-archive/vmware-tools-distrib/vmware-install.pl --default;
```

Comment:
Probably need to add a check here like we have in other scripts for the vmware version and set the option as appropriate.